### PR TITLE
Updated Professionalism/professionalism.md

### PR DESCRIPTION
**Pull request by tyagipr**
issue#570 
https://github.com/Techtonica/curriculum/blob/master/professionalism/professionalism.md
@manaswinidas

### CODE REVIEW

**Comment by manaswinidas** [comment]
@tyagipr I suppose this looks very similar to #617

**Comment by tyagipr** [comment]
yeah #617

**Comment by tyagipr** [comment]
now after changes  #620 @manaswinidas

**Comment by manaswinidas** [comment]
@tyagipr I guess you can push changes to a PR by pushing more commits to the same branch. This prevents duplication of efforts and prevents a lot of confusion.

**Comment by tyagipr** [comment]
I m sorry, i will not do this mistake again, this is my first time when i am working on open source.

**Comment by manaswinidas** [comment]
@tyagipr No worries! I totally get it. No need to panic and you needn't be sorry. :smiley: There is always a first time. You learn from your own mistakes. That's how you grow!

**Comment by tyagipr** [comment]
thanks for your support @manaswinidas

**Comment by tyagipr** [comment]
@manaswinidas will i get marks for this single file or when i complete all the links?

**Comment by alodahl** [comment]
@tyagipr you just have to do 1 :)

File: professionalism/professionalism.md [hunk 1]
```diff
@@ -39,15 +39,53 @@ Techtonica is a simulated work environment representative of common tech industr
 - Standards for the work day itself
```
Comment (alodahl) [inline professionalism/professionalism.md]: Change line 41 to just “Materials”.

File: professionalism/professionalism.md [hunk 2]
```diff
@@ -39,15 +39,53 @@ Techtonica is a simulated work environment representative of common tech industr
...
+
+[Professionalism (video walkthrough of slides)](https://drive.google.com/open?id=1WOor8RnhFz8yJimUSGf_QRY1_aplHure)
+[Professionalism (slides)](https://docs.google.com/presentation/d/1kvpeT1HTNJKDmnZr9amMe8TRrU_mCD8kwDRGxbomTFs/edit?usp=sharing)
 [Being Professional at Work (video)](https://www.youtube.com/watch?v=QphclWBVzGQ)  
 [How to Deal with People You Don't Like at Work (video)](https://www.youtube.com/watch?v=Pm8kU37u0Ho)  
 [Top 10 Tips for Early Career Professionals (video)](https://www.youtube.com/watch?v=bx6HTcQ8i2U)  
 
 ### Lesson
 
-[Professionalism (video walkthrough of slides)](https://drive.google.com/open?id=1WOor8RnhFz8yJimUSGf_QRY1_aplHure)
```
Comment (alodahl) [inline professionalism/professionalism.md]: Add line about working through everything in the “materials” section

**Pull request merged**